### DISCUSSION TITLE
MsSQL connection failure [Oct 08]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4111] [Discord Threads]
> 🙋‍♂️ **junaidbilal**  
## Facing his issue while connecting mssql  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1025561629116616754/Screenshot_20221001_051426.jpg)

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Can you provide more details about your setup & environment?   
####    
#### Able to connect to DB using same credentials outside NocoDB?   
#### Also, it appears on an older release. Can you upgrade to v0.97.0 and try.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1025561629380845608)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
